modules/configuration/php/configuration.class.inc

```diff
@@ -69,10 +69,47 @@ class Configuration extends \NDB_Form
         $this->tpl_data['instruments']     = $instruments;
         $this->tpl_data['sandbox']         = $config->getSetting("sandbox");
         $this->tpl_data['scan_types']      = $scan_types;
-        $this->tpl_data['lookup_center']   = array(
-            ''            => '',
-            'PatientID'   => 'PatientID',
-            'PatientName' => 'PatientName',
+        $DB = \NDB_Factory::singleton()->database();
+
+        $this->tpl_data['parentMenuItems'] = $this->_getParentConfigLabels();
+        $this->tpl_data['config']          = $this->_getConfigSettingTree();
+        $this->tpl_data['sandbox']         = \NDB_Factory::singleton()
+            ->config()
+            ->getSetting("sandbox");
+
+        // Build dropdown menus
+        $this->tpl_data['instruments']   = \Utility::toDropdownArray(
+            \Utility::getAllInstruments()
+        );
+        $this->tpl_data['scan_types']    = \Utility::toDropdownArray(
```

Comment:
I dont think this is equivalent to what was there before. the previous array used the scan ID from the database as key, this will use sequential keys. you can use the pselectColWithIndexKey and modify your `toDropdownArray` to accept associative arrays.